runtime/src/bank.rs

```diff
@@ -10611,6 +10624,8 @@ pub(crate) mod tests {
             ))
         );
         assert_eq!(bank.transaction_count(), 1);
+        assert_eq!(bank.executed_transaction_count(), 2);
```

Comment:
It's confusing that `executed_transaction_count()` is greater than the accumulated `transaction_count()`. Could you add a unit test for your new method that covers pre- and post-activation of `bank_transaction_count_fix`?

Reply:
Done